src/components/Loader.tsx

```diff
@@ -0,0 +1,21 @@
+import cn from 'classnames';
+import React from 'react';
+
+type Props = {
+  isLoading: number[];
+  id: number;
+};
+
+export const Loader: React.FC<Props> = ({ isLoading, id }) => {
+  return (
+    <div
+      data-cy="TodoLoader"
+      className={cn('modal overlay', {
+        'is-active': isLoading.includes(id),
```

Comment:
```suggestion
type Props = {
  isActive: boolean;
};

export const Loader: React.FC<Props> = ({ isActive }) => {
  return (
    <div
      data-cy="TodoLoader"
      className={cn('modal overlay', {
        'is-active': isActive,
```
The loader shouldn't know anything about id and loading todo ids, just show the loader